tasks/codex/edm.py

```diff
@@ -618,3 +618,49 @@ def fix_pending_sightings(context):
                 ags.config['sighting']['encounters'][i]['customFields'] = ecfv
         ags.config = ags.config
     db.session.flush()
+
+
+# note: will overwrite existing value currently
+@app_context_task(
+    help={
+        'cfguid': 'GUID of the customField',
+        'filename': 'filename to read data from (tab-delimited: model_guid, value)',
+    }
+)
+def custom_field_import(context, cfclass, cfguid, filename):
+    """
+    Import customField values from a file
+    """
+    from app.modules.encounters.models import Encounter
+    from app.modules.individuals.models import Individual
+    from app.modules.sightings.models import Sighting
+    from app.modules.site_settings.helpers import SiteSettingCustomFields
+
+    defn = SiteSettingCustomFields.get_definition(cfclass, cfguid)
+    if not defn:
+        raise ValueError(f'{cfclass}/{cfguid} invalid')
+
+    cls = None
+    if cfclass == 'Encounter':
+        cls = Encounter
+    elif cfclass == 'Sighting':
+        cls = Sighting
+    elif cfclass == 'Individual':
+        cls = Individual
```

Comment:
Sorry if this is unsolicited (was browsing through the repo and found the PR) 

A few alternatives to consider: 

Use a dict to maintain the mapping between str and the class type. And it can be even go into a constants.py or something. 
I.e.
```
CF_CLASS_TYPES = {
  ‘Encounter’: Encounter,
 …
}
```
That way the look ups are clean:

```
cls = CF_CLASS_TYPES[cfclass]
```